### DISCUSSION TITLE
[M-Z] (20 Gennaio 2020) Esercizio 6

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#456] [Esami & Esoneri M-Z]
## Traccia

<img width="931" alt="image" src="https://github.com/user-attachments/assets/feb9bfac-39be-4ceb-8910-daf2f424aabb">

**Comment by rimaout** [comment]
<img width="523" alt="image" src="https://github.com/user-attachments/assets/6ff65583-8b90-485d-9f92-e89c4f43e70b">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200120.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200120.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/929891bf-32a8-4959-a230-62ef6d705a20)
